extras/refresh-utils/src/main/java/com/palantir/remoting2/ext/refresh/RefreshableProxyInvocationHandler.java

```diff
@@ -54,7 +55,11 @@ private RefreshableProxyInvocationHandler(Refreshable<R> refreshable, Function<R
     protected Object handleInvocation(@Nonnull Object proxy, @Nonnull Method method, @Nonnull Object[] args)
             throws Throwable {
         updateDelegate();
-        return method.invoke(delegate, args);
+        try {
+            return method.invoke(delegate, args);
+        } catch (InvocationTargetException e) {
+            throw e.getCause();
```

Comment:
Is what guaranteed to be non-null? e or getCause()?

If it's the former, no. 

If it's the latter. According to the Javadocs yes, however, in reality, the default constructor for ITE is protected, and the only other constructors technically allow a null target, but practically, this is basically an extremely illegal usage of the API, so I wouldn't expect it to ever occur in practice.